x-pack/legacy/plugins/ml/common/types/categories.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export type CategoryId = number;
+
+export interface Category {
+  job_id: string;
```

Comment:
We need to refactor the job interfaces, currently they are located inside the wizard code but should be moved here to common.
This hasn't happened so far because there is an existing partial job interface that will need to be replaced.
For now I think we should leave this as is, and look to clean up these interfaces in the future.